node/gum/proc-macro/src/lib.rs

```diff
@@ -43,6 +43,27 @@ pub fn warn(item: proc_macro::TokenStream) -> proc_macro::TokenStream {
 	gum(item, Level::Warn)
 }
 
+/// Print a warning or debug level message depending on their frequency
```

Comment:
I would expand this to also include more details and examples as well as limitations.